### DISCUSSION TITLE
Ask to auto install the dependencies

### DISCUSSION THREAD

**Discussion by j-d-carmichael** [acr-lfr/generate-it#112] [Ideas]
When there are dependency changes between the installed package.json file and the json file from the git repo of the tpl - the core will spit out the npm i command with all the new dep versions.

However.. you have to copy and paste them all in - would be cleaner if the core just asked then installed them. 

Maybe an additional flag to opt into this functionality.. but maintaining dependency updates over a load of apis by hand is a pain in the arse!

@acrontum-sanchez   @p-mcgowan

**Comment by p-mcgowan** [comment]
Yeah that would be cool - definitely something that would need an opt-in though as a CLI blocking feature might surprise people.

Sounds like a feature ticket to me :D

**Comment by j-d-carmichael** [comment]
You can cover discussions to issues which is nice Gh feature

**Reply by j-d-carmichael** [reply]
On desktop at least

**Reply by p-mcgowan** [reply]
ah sweet - just like the project board. that's cool
